sway/sway.5.scd

```diff
@@ -770,17 +770,19 @@ Mark all Firefox windows with "Browser":
 [class="Firefox"] mark Browser
 ```
 
+You may like to use swaymsg -t get_tree for finding the values of these properties in practice for your applications.
+
 The following attributes may be matched with:
 
 *app_id*
 	Compare value against the app id. Can be a regular expression. If value is
 	\_\_focused\_\_, then the app id must be the same as that of the currently
-	focused window.
+	focused window. _app_id_ are specific to Wayland applications.
```

Comment:
The similar comment on *class* is fine, but since sway is a Wayland compositor we needn't document that it has Wayland-specific behavior.

Reply:
I think it's still useful to say it's only for Wayland applications, otherwise users will expect app_id to work with X11 clients too.

Reply:
I wrote this to help users understand that if they cannot find _class_ using swaymsg -t get_tree is probably because it's a Wayland application. It something that trouble me when I tried to use for_window. I think that since this is a pretty new behaviour (especially for users coming from i3) it could be good to document it.

Is it not relevant or is it not explicit enough?

Reply:
Nevermind, this is fine as-is (not including the 80 col issue)